pkgs/top-level/all-packages.nix

```diff
@@ -1493,7 +1493,24 @@ with pkgs;
 
   pcsxr = callPackage ../applications/emulators/pcsxr { };
 
-  ppsspp = libsForQt5.callPackage ../applications/emulators/ppsspp { };
+  ppsspp-qt = libsForQt5.callPackage ../applications/emulators/ppsspp {
+    pname = "ppsspp-qt";
+    enableVulkan = false; # https://github.com/hrydgard/ppsspp/issues/11628
+  };
+
+  ppsspp-sdl = callPackage ../applications/emulators/ppsspp {
+    pname = "ppsspp-sdl";
```

Comment:
`pname` should not be a "visible parameter" to all-packages.nix.

Instead you can programatically change it inside the derivation, like we do with cmake:

https://github.com/NixOS/nixpkgs/blob/a50594822c73de196d148991eb110f6484fd123b/pkgs/development/tools/build-managers/cmake/default.nix#L35